auth-web/src/models/eft-transaction.ts

```diff
@@ -0,0 +1,43 @@
+export interface EFTTransaction {
+  id: number
+  shortNameId: number
+  depositAmount: number
+  depositDate: string
+  transactionDate: string
+}
+
+export interface EFTTransactionFilterParams {
+  pageNumber?: number
+  pageLimit?: number
+}
+
+export interface EFTTransactionListResponse {
+  items: EFTTransaction[]
+  limit: number
+  page: number
+  total: number
+  remainingCredit: number
+}
+
+export interface EFTTransactionState {
+  filters: EFTTransactionFilterParams
+  loading: boolean
+  results: EFTTransaction[]
+  totalResults: number
+}
+
+export interface EFTShortnameResponse {
```

Comment:
Any of these optional? 

Might want to add in ? after optional fields.. EX. ones that can be null

Reply:
Updated